main/options/options.cc

```diff
@@ -311,7 +311,6 @@ cxxopts::Options buildOptions() {
     options.add_options("advanced")("watchman-path",
                                     "Path to watchman executable. Defaults to using `watchman` on your PATH.",
                                     cxxopts::value<string>()->default_value(empty.watchmanPath));
-    options.add_options("advanced")("enable-experimental-lsp-hover", "Enable experimental LSP feature: Hover");
```

Comment:
Note that this will break people who pass this flag.
I'm fine with that(flag is experimental), but this is worth noting in commit message.

Reply:
I will edit the merge commit message to note the issue.